api/src/test/java/org/openmrs/module/metadatadeploy/handler/impl/ConceptDeployHandlerTest.java

```diff
@@ -56,223 +56,192 @@
-                        hasProperty("localePreferred", is(false)),
-                        hasProperty("conceptNameType", nullValue()),
-                        hasProperty("tags", containsInAnyOrder(Arrays.asList(
-                                hasProperty("tag", is("refapp-preferred"))
-                        ))))));
-        assertThat(updated.getNames(true).size(), is(updated.getNames(false).size() + 1));
-        assertThat(updated.getDescriptions().size(), is(1));
-        assertThat(updated.getDescription(), allOf(
-                hasProperty("uuid", is("75bcd12e-cdc4-11e4-9dcf-b36e1005e77b")),
-                hasProperty("description", is("How the patient describes it")),
-                hasProperty("locale", is(Locale.ENGLISH))));
-        assertThat(updated.getConceptMappings(), containsInAnyOrder(
-                allOf(
-                        hasProperty("uuid", is("a361ab43-065b-40da-89a9-3a095de359af")),
-                        hasProperty("conceptMapType", hasProperty("name", is("same-as"))),
-                        hasProperty("conceptReferenceTerm", allOf(
-                                hasProperty("code", is("422625006")),
-                                hasProperty("conceptSource", hasProperty("uuid", is("j3nfjk33-639f-4cb4-961f-1e025b908433")))
-                        ))
-                ),
-                allOf(
-                        hasProperty("uuid", is("new-mapping-uuid")),
-                        hasProperty("conceptMapType", hasProperty("name", is("same-as"))),
-                        hasProperty("conceptReferenceTerm", allOf(
-                                hasProperty("code", is("1234567890")),
-                                hasProperty("conceptSource", hasProperty("uuid", is("j3nfjk33-639f-4cb4-961f-1e025b908433")))
-                        ))
-                )));
-
-        // Check everything can be persisted
-        Context.flushSession();
-    }
-
-    @Test
-    public void testUpdateNumeric() throws Exception {
-        String weightUuid = "c607c80f-1ea9-4da3-bb88-6276ce8868dd";
-        deployService.installObject(new ConceptNumericBuilder(weightUuid)
-                .datatype(conceptService.getConceptDatatypeByName("Numeric"))
-                .conceptClass(existing(ConceptClass.class, "a82ef63c-e4e4-48d6-988a-fdd74d7541a7")) // doesn't matter which
-                .name(new ConceptName("WEIGHT (KG)", Locale.ENGLISH))
-                .lowAbsolute(0d)
-                .hiAbsolute(null) // change this just to see if it works
-                .precise(true)
-                .build());
-
-        ConceptNumeric updated = (ConceptNumeric) conceptService.getConceptByUuid(weightUuid);
-        assertThat(updated.getLowAbsolute(), is(0d));
-        assertThat(updated.getHiAbsolute(), nullValue());
-        assertThat(updated.getPrecise(), is(true));
-        assertThat(updated.getName().getName(), is("WEIGHT (KG)"));
-    }
-
-    @Test
-    public void testRetire() throws Exception {
```

Comment:
Did you make any changes in this method? If so, did you rewrite the whole method?

Reply:
i did not make any changes here and have removed the file that i had pushed